tests/python_package_test/test_dask.py

```diff
@@ -291,6 +278,8 @@ def test_regressor(output, client, listen_port):
     if output != 'dataframe':
         s1 = r2_score(dy, p1)
     p1 = p1.compute()
+    p2_local_predict = dask_regressor.to_local().predict(X)
+    s2_local_predict = dask_regressor.to_local().score(X, y)
```

Comment:
```suggestion
    p2_local = dask_regressor.to_local().predict(X)
    s2_local = dask_regressor.to_local().score(X, y)
```

Please simplify these names a bit (and elsewhere in the PR) by removing the "_predict" suffix